addons/website/static/src/js/content/body_manager.js

```diff
@@ -1,10 +1,10 @@
 odoo.define('web_editor.BodyManager', function (require) {
 'use strict';
 
-var weContext = require('web_editor.context');
-var rootWidget = require('web_editor.root_widget');
+var weContext = require('website.context');
```

Comment:
we is for web_editor, rename the variable too I suppose ? Same everywhere

Reply:
:+1: